src/model/transaction/removeRangeFromContentState.js

```diff
@@ -62,7 +62,7 @@ function removeRangeFromContentState(
     .toSeq()
     .skipUntil((_, k) => k === startKey)
     .takeUntil((_, k) => k === endKey)
-    .concat(Immutable.Map([[endKey, null]]))
+    .filter((v, k) => k !== endKey)
```

Comment:
Unfortunately this doesn't provide the same functionality as the previous version. Going to silence the error and open an issue to fix some of these 'flowfixme' comments.